main/src/cgeo/geocaching/storage/LocalStorage.java

```diff
@@ -449,6 +450,12 @@ protected Integer doInBackgroundInternal(final Void[] params) {
                 }
             }
 
+            //delete theme files from outdated internal theme file dir
```

Comment:
IMHO no need to implement a migration logic for nightly users only.
But might be good to be prepared if we need to make changes in future.

Reply:
Really no migration? Some nightly users seem to care a lot about pollution of internal storage, there is no way to get outdated files out of it unless we do it in migration.

Reply:
mmh. The nightly user should be able to access `/Android/data/cgeo.geocaching/...`. But you might of course keep it in, I just wanted to make live easier as I think we only should really care about migrations between release versions as nightlies are always experimental.

Reply:
Ok, maybe that was overkill. I can put it out again next time I am able to code, or we just merge it (it doesn't harm either way)

Reply:
Lets keep it in...as you say. Does not harm.